compiler/rustc_codegen_gcc/src/intrinsic/mod.rs

```diff
@@ -74,8 +74,14 @@ fn get_simple_intrinsic<'gcc, 'tcx>(
         sym::fabsf64 => "fabs",
         sym::minnumf32 => "fminf",
         sym::minnumf64 => "fmin",
+        sym::minimumf32 => "fminimumf",
+        sym::minimumf64 => "fminimum",
+        sym::minimumf128 => "fminimuml",
         sym::maxnumf32 => "fmaxf",
         sym::maxnumf64 => "fmax",
+        sym::maximumf32 => "fmaximumf",
+        sym::maximumf64 => "fmaximum",
+        sym::maximumf128 => "fmaximuml",
```

Comment:
Do these need to be names that GCC recognizes or can they be any function (cc @antoyo)? It would be better to lower to `fminimumf128` and `fmaximumf128`, `*l` functions are absolutely cursed and I'd rather avoid even mentioning them. Preexisting `copysignl` might as well also get updated too if we can do this.

We provide these in builtins so availability isn't a concern https://github.com/rust-lang/compiler-builtins/blob/a4c748f72a1dce652cc3e41c3a8425731bd1519a/compiler-builtins/src/math/mod.rs#L92-L101.

Reply:
This function `get_simple_intrinsic` returns a `gccjit::Function`, so it could be an "extern" function if you wanna call some existing function defined elsewhere.

Reply:
As asked, I switched to `fminimumf128` and `fmaximumf128`.